app/controllers/answers_controller.rb

```diff
@@ -0,0 +1,60 @@
+class AnswersController < ApplicationController
+  before_action :user_auth, only: %i[create update destroy]
+  before_action :set_answer, only: %i[update destroy]
+  before_action :set_message, only: %i[update destroy]
+  before_action :check_question_presence, only: %i[create update destroy]
+
+  def index
+    render json: Answer.all, status: :ok
+  end
+
+  def create
+    answer = Answer.new(answer_params)
+    answer.user_id = @current_user.id
+    answer.question_id = question.id
```

Comment:
checkout ".merge" method on hash objects and do this creation of answer in one line.